### PR TITLE
Refactor CSS files in src/screens/SubTags

### LINKED ISSUE #2526: Refactor CSS files in src/screens/SubTags
# Refactor CSS files in src/screens/SubTags

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by amitb0ra**
**What kind of change does this PR introduce?**

refactoring

**Issue Number:**

Fixes #2526

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

image

There is some issue: "Cannot find module '@testing-library/dom' from 'node_modules/@testing-library/react/dist/pure.js'" but that is not relevant here.

**If relevant, did you update the documentation?**

No

**Summary**

The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern.

**Does this PR introduce a breaking change?**

No

**Other information**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Updated pull request template to include detailed guidance for contributors, including prompts for change types and documentation updates.

- **Style**
	- Removed the `SubTags.module.css` file and incorporated its styles into the existing CSS structure for better organization.
	- Enhanced styles in `app.module.css` for improved layout and responsiveness, including new media queries and interactive elements.

- **Bug Fixes**
	- Corrected syntax issues in `jest.config.js` for proper configuration.

### CODE REVIEW

**Comment by amitb0ra** [comment]
Hi, I noticed that many unrelated files have been included in this pull request. Additionally, I’d like to join the organization’s Slack workspace, but it seems to require an PalisadoesFoundation email. Could you please let me know how I can join without one or if there’s an alternative process?

**Comment by palisadoes** [comment]
1. Please submit your PRs against our `develop-postgres` branch. 
2. Use a local branch that is not `develop`. Best practices requires a non `develop*` name. This will make it easier for us to review.
    1. Make the basis of this local branch be the repo's `develop-postgres` branch. 
4. You have included files that are not required for the issue to be resolved.

Closing

**Comment by palisadoes** [comment]
The slack link now works. Thanks for the reminder

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ Fixes #<!--Add related issue number here.-->
 
 **Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
```
Comment (palisadoes) [inline .github/pull_request_template.md:62]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

**Pull request closed without merging**